Extension/package.nls.json

```diff
@@ -26,6 +25,7 @@
     "c_cpp.command.RunCodeAnalysisOnOpenFiles.title": "Run Code Analysis on Open Files",
     "c_cpp.command.RunCodeAnalysisOnAllFiles.title": "Run Code Analysis on All Files",
     "c_cpp.command.ClearCodeAnalysisSquiggles.title": "Clear Code Analysis Squiggles",
+    "c_cpp.command.BuildAndDebugFile.title": "Build and Debug File",
```

Comment:
Does this command still show up in the Command Palette?  It might still need to say "Active" for context.